doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -185,22 +186,22 @@
 
     ```rust
     // old
-    let services = zero_copy::Service::list()?;
-    let services = zero_copy::Service::list_with_custom_config(Config::global_config())?;
+    let services = ipc::Service::list()?;
+    let services = ipc::Service::list_with_custom_config(Config::global_config())?;
```

Comment:
Just wondering whether the example of the old code should still have `zero_copy` as name.